### PR TITLE
Refactored Password Manager in OOPS way

### PR DESCRIPTION
**Pull request by Pratik2803**
# Refactored Password Manager  in OOPS way as suggested in morning
# Instead of text file start saving saving data in JSON file

### CODE REVIEW

File: password-manager-start/password_generator.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from random import choice, shuffle, randint
+from tkinter import END
+
+LETTERS = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u',
```
Comment (snaps2k) [inline password-manager-start/password_generator.py:4]: Put them in seperate file or a constant file, if they are to be used in multiple places, if not move them inside class. Better to keep class functionality in this class.

Same comment if for other class too.

File: password-manager-start/ui.py [hunk 2]
```diff
@@ -13,9 +13,12 @@
 # Buttons Text
 GEN_PASS_BTN = 'Generate Password'
 ADD_BTN = 'ADD'
+SEARCH_BTN = 'Search'
```
Comment (snaps2k) [inline password-manager-start/ui.py:16]: Same issue as other comment

File: password-manager-start/password_generator.py (hunk 1 above)
Comment (Pratik2803) [reply password-manager-start/password_generator.py]: Refer latest commit

File: password-manager-start/ui.py (hunk 2 above)
Comment (Pratik2803) [reply password-manager-start/ui.py:16]: Please ignore this comment I will take care in future

**Pull request merged**